src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -175,6 +175,35 @@ sequences using different contrast enhanced images. The label is the name of the
 contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
 JSON file, with the same label.
 
+#### The `part` entity
+
+This entity is used to indicate which component of the complex representation of the MRI signal is represented in voxel data.
+The `part-<label>` key/value pair is associated with the DICOM tag `0008,9208`.
+Allowed label values for this entity are `phase`, `mag`, `real` and `imag`, which are typically used in `mag/phase` or `real/imag` pairs.
+For example:
+
+```Text
+sub-01_part-mag_T1w.nii.gz
+sub-01_part-mag_T1w.json
+sub-01_part-phase_T1w.nii.gz
+sub-01_part-phase_T1w.json
+```
+
+Phase images MAY be in radians or in arbitrary units.
+The sidecar JSON file MUST include the units of the `phase` image.
+The possible options are: `radians` or `a.u.` for grayscale images.
+For example:
+
+sub-01_part-phase.json
+
+```Text
+{
+   "Units": "radians"
+}
+```
+
+If the `part` key/value pair is absent from a filename, the image SHOULD be a magnitude image.
```

Comment:
This feels like a strange directive. Does it permit me to have:

```
sub-01_T1w.nii.gz
sub-01_part-phase_T1w.nii.gz
```

I might instead treat this as `run-`, which we say may be omitted if only one run is provided.

```suggestion
When there is only a magnitude image of a given type, the `part` key MAY be omitted.
```